examples/search-essential-information/Search essential information.md

```diff
@@ -0,0 +1,24 @@
+# Search Essential Information
```

Comment:
Can we use sentence case in example docs as well?

Reply:
Are we using sentence cases for titles as well?

Reply:
Yeah, it seems so
https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/tree/master/examples/process-mails

Reply:
and check this, https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/examples/reply-mails/Automated%20Email%20Responses.md :)
Shall we confirm this?

Reply:
But in most cases, it is using sentence case, but it is not conventional to use sentence case for a title, will verify this

Reply:
will, go with sentence case

Reply:
Yeah. @NipunaRanasinghe what would be the better way here?

Reply:
We should follow sentence case as it is the recommendation from our docs team. I will update gmail examples references.